sample_jsons/asff_mapper/sample_input_jsons/asff_sample.json

```diff
@@ -0,0 +1,145 @@
+            "Region": "us-east-1",
+            "GeneratorId": "arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0/rule/1.1",
+            "AwsAccountId": "123456789123",
+            "Types": [
+                "Software and Configuration Checks/Industry and Regulatory Standards/CIS AWS Foundations Benchmark"
+            ],
+            "FirstObservedAt": "2021-07-23T23:39:05.607Z",
+            "LastObservedAt": "2021-08-04T23:29:17.637Z",
+            "CreatedAt": "2021-07-23T23:39:05.607Z",
+            "UpdatedAt": "2021-08-04T23:29:15.639Z",
+            "Severity": {
+                "Product": 30,
+                "Label": "LOW",
+                "Normalized": 30,
+                "Original": "LOW"
+            },
+            "Title": "1.1 Avoid the use of the \"root\" account",
+            "Description": "The \"root\" account has unrestricted access to all resources in the AWS account. It is highly recommended that the use of this account be avoided.",
+            "Remediation": {
+                "Recommendation": {
+                    "Text": "For directions on how to fix this issue, please consult the AWS Security Hub CIS documentation.",
+                    "Url": "https://docs.aws.amazon.com/console/securityhub/standards-cis-1.1/remediation"
+                }
+            },
+            "ProductFields": {
+                "StandardsGuideArn": "arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0",
+                "StandardsGuideSubscriptionArn": "arn:aws:securityhub:us-east-1:123456789123:subscription/cis-aws-foundations-benchmark/v/1.2.0",
+                "RuleId": "1.1",
+                "RecommendationUrl": "https://docs.aws.amazon.com/console/securityhub/standards-cis-1.1/remediation",
+                "StandardsControlArn": "arn:aws:securityhub:us-east-1:123456789123:control/cis-aws-foundations-benchmark/v/1.2.0/1.1",
+                "aws/securityhub/ProductName": "Security Hub",
+                "aws/securityhub/CompanyName": "AWS",
+                "aws/securityhub/annotation": "Multi region CloudTrail with the required configuration does not exist in the account",
+                "Resources:0/Id": "arn:aws:iam::123456789123:root",
+                "aws/securityhub/FindingId": "arn:aws:securityhub:us-east-1::product/aws/securityhub/arn:aws:securityhub:us-east-1:123456789123:subscription/cis-aws-foundations-benchmark/v/1.2.0/1.1/finding/564b8d35-d697-475e-8a02-dfb23552dc24"
+            },
+            "Resources": [
+                {
+                    "Type": "AwsAccount",
+                    "Id": "AWS::::Account:123456789123",
+                    "Partition": "aws",
+                    "Region": "us-east-1"
+                }
+            ],
+            "Compliance": {
+                "Status": "FAILED",
+                "StatusReasons": [
+                    {
+                        "ReasonCode": "CLOUDTRAIL_MULTI_REGION_NOT_PRESENT",
```

Comment:
I am bit concerned here that the fail reason does not seem to align with the title.
"Title": "1.1 Avoid the use of the \"root\" account",

Could we acquire a larger sample with more Compliance statuses.

Reply:
I was trying to pare down the sample size since that was a request from the scoutsuite mapper.  However, I've now regenerated it with all ~15 failures from Aaron's account instead of just the 2 I cut it down to.

The status reasons sometimes seem to contain spurious information.  In the interest of visibility, I just push it all through to the user.

Reply:
Changed sample file to include all the findings from Aaron as well as one from scrubbed sponsor data.